sdks/builds/llvm.mk

```diff
@@ -52,11 +51,15 @@ build-custom-llvm-$(1):
 	cmake --build $$(TOP)/sdks/builds/llvm-$(1)
 
 package-llvm-$(1):
-	cmake --build $$(TOP)/sdks/builds/llvm-$(1) --target install
+	$$(MAKE) -C $$(TOP)/llvm -f Makefile.am install-llvm \
+		LLVM_PATH="$(LLVM_SRC)" \
+		LLVM_BUILD="$$(TOP)/sdks/builds/llvm-$(1)" \
+		LLVM_PREFIX="$$(TOP)/sdks/out/llvm-$(1)" \
+		$$(if $$(llvm-$(1)_CMAKE_ARGS),LLVM_CMAKE_ARGS="$$(llvm-$(1)_CMAKE_ARGS)")
 
 .PHONY: clean-llvm-$(1)
 clean-llvm-$(1):
-	rm -rf .stamp-llvm-$(1)-toolchain .stamp-llvm-$(1)-configure $$(TOP)/sdks/builds/llvm-$(1) $$(TOP)/sdks/builds/llvm-$(1).config.cache $$(TOP)/sdks/out/llvm-$(1)
+	rm -rf .stamp-llvm-$(1)-toolchain .stamp-llvm-$(1)-configure && $$(MAKE) clean -C $$(TOP)/llvm
```

Comment:
You should remove the `.stamp-llvm-$(1)-toolchain` and `.stamp-llvm-$(1)-configure` targets at respectively https://github.com/mono/mono/pull/8792/files#diff-e19948cc4c89121d1aa1068fd3d60b01R40 and https://github.com/mono/mono/pull/8792/files#diff-e19948cc4c89121d1aa1068fd3d60b01R43 since all configure and build related steps need to be done in `llvm/`.

Reply:
Could also please integrate the building of llvm with mxe as a different target in `llvm/`.